pliers/extractors/api/awsReko_test.py

```diff
@@ -0,0 +1,82 @@
+'''
+Extractors that interact with the AWS Rekognition API.
+'''
+
+from pliers.stimuli.image import ImageStim
+from pliers.extractors.base import Extractor, ExtractorResult
+import numpy as np
+import pandas as pd
+import boto3
+from pliers.utils import attempt_to_import, verify_dependencies
+
+aws_rekognition_client= attempt_to_import('boto3')
+
+
+class AwsRekognitionExtractor():
+    
+    def __init__(self,profile_name='default', region_name='us-east-1' ):
+            verify_dependencies(['boto3'])
+            self.session = boto3.Session(profile_name='user2')
```

Comment:
Assuming the user's credential file is set up correctly, we should be able to just call `boto3.client('rekognition')`, I think. We should probably handle both cases—i.e., (a) where `profile_name` is `None`, in which case you just initialize directly, and (b) where `profile_name` is passed, in which case you initialize the session explicitly.